vm/src/builtins/bytes.rs

```diff
@@ -366,13 +366,31 @@ impl PyBytes {
     }
 
     #[pymethod]
-    fn lstrip(&self, chars: OptionalOption<PyBytesInner>) -> Self {
-        self.inner.lstrip(chars).into()
+    fn lstrip(
+        zelf: PyRef<Self>,
+        chars: OptionalOption<PyBytesInner>,
+        vm: &VirtualMachine,
+    ) -> PyRef<Self> {
+        let stripped = zelf.inner.lstrip(chars);
+        if stripped == zelf.as_bytes().to_vec() {
```

Comment:
Does this work?

```suggestion
        if stripped == zelf.as_bytes() {
```

Reply:
I think `&stripped == zelf.as_bytes()` works, but still stripped is `Vec<u8>`.